supervisor/misc/filter.py

```diff
@@ -34,6 +34,8 @@ def filter_data(coresys: CoreSys, event: dict, hint: dict) -> dict:
     if not coresys.config.diagnostics or not coresys.core.supported or dev_env:
         return None
 
+    event.setdefault("extra", {}).update({"os.environ": dict(os.environ)})
```

Comment:
Can we add this to the "context" instead as a new entry in that dict?

And maybe also create a new dict from it, so we can filter out any keys containing "token"

Reply:
Because it's mach with the sys.args feature on the event.

We don't have any kind of trust or secrets on the env for supervisor

Reply:
![image](https://user-images.githubusercontent.com/15338540/91633862-eeba6200-e9eb-11ea-8db8-5268dff27970.png)